templates/energyplus/templates/ref-sys.pxt

```diff
@@ -160,5 +161,6 @@ EnergyManagementSystem:ProgramCallingManager,
 	
 EnergyManagementSystem:Program,	
 CondTempControl<%= ref_system_name %>,       !- Name
-SET CondensingTempOverride<%= ref_system_name %> = T_sensor_<%= ref_system_name %> + <%= cond_sct_delta %>;  !- Program Line 1
+SET CondensingTempOverride<%= ref_system_name %> = T_sensor_<%= ref_system_name %> + <%= cond_sct_delta %>,  !- Program Line 1
+SET CondensingTempOverride<%= ref_system_name %> = @Max <%= cond_min_temp %> CondensingTempOverride<%= ref_system_name %>;  !- Program Line 2
```

Comment:
If floating condenser temperature (`refsys_cond_floating` = true), in EMS logic for minimum condensing temperature, set a lower bound (using the same parameter used above).